components/molecule/notification/src/index.js

```diff
@@ -109,7 +109,8 @@ class MoleculeNotification extends Component {
       position,
       showCloseButton,
       effect,
-      text
+      text,
```

Comment:
`children` can be anything, also text, so... does it still make sense?

Reply:
I mean, prop `text` could be removed, so that you should create a major version.

Reply:
Totally agree

Reply:
While I understand the comment, and it make sense, I would advise trying to not use a major for this. Keeping compatibility with previous versions and adding the `children` is enough for adding the feature and it will let projects to adopt this change seamlessly without having to move to a major.

I rather would prefer to use children as default and add a deprecated notice for the usage of `text` but do it as a minor still.

Reply:
The idea of doing a major... is to clean up code. Of course the major is not needed and we could make it this changes as a minor. But IMHO, we are creating a _"technical debt"_ that we could solve doing the major now.

In addition, then we should add a `wrapper` for this `content` that renders the `children` because we should apply some spacing between `text` and the `content`. "Too much" for evading to create a major in my opinion. What do you think guys? @miduga @kikoruiz @JoanClaret

Reply:
The code to maintain and the technical debt is negligible against the lack of support for actual consumers of the component and the problems and work that this could originate. Creating a major version of a component without even offering a deprecation period is not great and it goes against the idea behind SUI Components for creating a stable set of components. Even that, this decision can't be based on two or three approves in a PR that only wanted to add a feature.

**If we create a major version now:**
- Projects that are already using this component will need to move to the latest version to get fixes and new features.
- If they want to not impact performance, they will need to move all components at once in order to avoid loading two versions of the same component.

IMHO we should be very careful with majors and iterate over actual versions as much as possible. **Does it mean are we not going to release a major never? No.** But we have to be responsible with this and only create majors when they're strictly necessary or after having a bag of deprecations after some time of working on them. The problem never should be creating technical debt, the problem would be not addressing never them.

**What we could do instead?**
- Deprecate the usage of the prop but maintaining compatibility with actual usages as this PR is proposing.
- Require the `children` prop in PropTypes and make `text` optional. Add in the description of `text` prop that the prop is deprecated.
- Share in our channels this deprecation notice and give info in order to move to the new version.
- Give a period of grace in order to users of the component to move to the new prop without hassle.
- If technical debt is produced, comment it over and create issues if necessary, explaining it in order to address in the next major.

Reply:
Mhh.. You sold it right. I agree then that this could be the best solution for everyone